test/e2e/pivoting_test.go

```diff
@@ -252,13 +253,21 @@ func removeIronicContainers() {
 		Expect(err).To(BeNil(), "Unable to delete the container %s: %v", container, err)
 	}
 }
+
 func removeIronicDeployment() {
 	deploymentName := os.Getenv("NAMEPREFIX") + "-ironic"
 	ironicNamespace := os.Getenv("IRONIC_NAMESPACE")
 	err := bootstrapClusterProxy.GetClientSet().AppsV1().Deployments(ironicNamespace).Delete(ctx, deploymentName, metav1.DeleteOptions{})
 	Expect(err).To(BeNil(), "Failed to delete Ironic from the source cluster")
 }
 
+func removeIronicDeploymentOnTarget() {
+	deploymentName := os.Getenv("NAMEPREFIX") + "-ironic"
+	ironicNamespace := os.Getenv("IRONIC_NAMESPACE")
+	err := targetCluster.GetClientSet().AppsV1().Deployments(ironicNamespace).Delete(ctx, deploymentName, metav1.DeleteOptions{})
+	Expect(err).To(BeNil(), "Failed to delete Ironic from the source cluster")
```

Comment:
```suggestion
	Expect(err).To(BeNil(), "Failed to delete Ironic from the target cluster")
```

Reply:
This will be covered as part of #390 when I will backport it to release-0.5, so I am fine merging this as is.

Reply:
*target

Reply:
fixed